dvc/commands/dag.py

```diff
@@ -127,6 +155,18 @@ def add_parser(subparsers, parent_parser):
         default=False,
         help="Print DAG with .dot format.",
     )
+    dag_parser.add_argument(
+        "--mermaid",
+        action="store_true",
+        default=False,
+        help="Print DAG with mermaid format.",
+    )
+    dag_parser.add_argument(
+        "--md",
```

Comment:
Even though it's deprecated, `--show-md` flag would be nice till 3.0 for consistency. Also please set `dest=markdown` for consistency.

Here's how you can alias them:
https://github.com/iterative/dvc/blob/72d57f13bc8ad4389516ea5f22d65200799a8ac7/dvc/commands/params.py#L113-L120

You can add a test below for testing compatibility:
https://github.com/iterative/dvc/blob/72d57f13bc8ad4389516ea5f22d65200799a8ac7/tests/unit/command/test_compat_flag.py#L39